client/main.lua

```diff
@@ -82,13 +67,28 @@ RegisterNetEvent('qb-banking:openBankScreen', function()
     openAccountScreen()
 end)
 
--- Loop
+local BankControlPress = false
+ local function BankControl(variable)
+    CreateThread(function()
+        BankControlPress = true
+        while BankControlPress do
+            if IsControlPressed(0, 38) then
+                exports['qb-core']:KeyPressed()
+                if variable == "bank" then
```

Comment:
Since you only have a single use of the this function get rid of this, as its just a useless check.

Reply:
I was talking about the varaible == "bank" but github decided to screw me over and not highlight it

Reply:
I understood what you ment 😉